.circleci/config.yml

```diff
@@ -53,8 +53,8 @@ jobs:
           command: |
             source ${INIT_ENV}
             cd src
-            git clone --depth=1 https://github.com/usdot-fhwa-stol/CARMAUtils.git --branch CARMAUtils_1.3.0
-            git clone --depth=1 https://github.com/usdot-fhwa-stol/CARMAMsgs.git --branch CARMAMsgs_1.1.0
+            git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch feature/building_with_colcon
```

Comment:
Looks like this go messed up. Should be the system release or left as pointing to utils 1.3.0